### PR TITLE
Task/update installation guides with mqtt installation instructions paho and mosquitto

### PR DESCRIPTION
**Pull request by joaoGabriel55**

### CODE REVIEW

File: doc/manuals-ld/installation-guide-debian9.md [hunk 1]
```diff
@@ -221,7 +221,14 @@ make html
 make
 sudo make install
 
+
 # Python library
+
+# [ If you don't have pip installed: ]
```
Comment (kzangeli) [inline doc/manuals-ld/installation-guide-debian9.md]:
```suggestion
# If you don't have pip installed:
```

File: doc/manuals-ld/installation-guide-debian9.md [hunk 2]
```diff
@@ -221,7 +221,14 @@ make html
...
+sudo aptitude update
+sudo aptitude install python-pip
+pip --version
```
Comment (kzangeli) [inline doc/manuals-ld/installation-guide-debian9.md]: If we ask the user to issue a "pip --version" command, we should also tell them what should be the desired output of the command.

Either that, or just remove the version line.

**Review by kzangeli — CHANGES_REQUESTED** [review]

File: doc/manuals-ld/installation-guide-debian9.md (hunk 2 above)
Comment (joaoGabriel55) [reply doc/manuals-ld/installation-guide-debian9.md]: Okay

File: doc/manuals-ld/installation-guide-centos7.md [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+## Grant sudo permissions to your user (if you don't have)
+
+```bash
+visudo
+
+# On this line that contains: root ALL=(ALL) ALL, add your user
+<USER> ALL=(ALL) ALL
+```
+
+*Obs.: If you don't have internet access execute this command:*
```
Comment (kzangeli) [inline doc/manuals-ld/installation-guide-centos7.md:5]: That would be valid for each and every command of this guide ... :)

Let's remove it. The users must be able to figure this out by themselves

File: doc/manuals-ld/installation-guide-centos7.md [hunk 4]
```diff
@@ -17,7 +31,9 @@ To be installed via package manager:
 Tools needed for compilation and testing:
 
 ```bash
-sudo yum install make cmake gcc-c++ scons
+sudo yum install http://repo.okay.com.mx/centos/7/x86_64/release/okay-release-1-1.noarch.rpm
```
Comment (kzangeli) [inline doc/manuals-ld/installation-guide-centos7.md:27]: What is this?   okay, from Mexico?

People will wonder.
Add a short description on why this package is necessary

**Review by kzangeli — CHANGES_REQUESTED** [review]
Minor fixes

File: doc/manuals-ld/installation-guide-centos7.md (hunk 3 above)
Comment (joaoGabriel55) [reply doc/manuals-ld/installation-guide-centos7.md:5]: Okay

File: doc/manuals-ld/installation-guide-centos7.md (hunk 4 above)
Comment (joaoGabriel55) [reply doc/manuals-ld/installation-guide-centos7.md:27]: I will explain

**Pull request merged**